src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -1501,10 +1501,16 @@ class WellKnownTextGeometrySegment(BaseSegment):
         Sequence(
             OneOf("GEOMETRY", "GEOGRAPHY"),
             Bracketed(
-                Sequence(
-                    OneOf(*_geometry_type_keywords, "GEOMETRY", "GEOGRAPHY"),
-                    Ref("CommaSegment"),
-                    Ref("NumericLiteralSegment"),
+                OneOf(
+                    "POINT",
+                    Sequence(
+                        OneOf(*_geometry_type_keywords, "GEOMETRY", "GEOGRAPHY"),
+                        Sequence(
+                            Ref("CommaSegment"),
+                            Ref("NumericLiteralSegment"),
+                            optional=True,
+                        ),
+                    ),
```

Comment:
I think this can be further simplified because `POINT` is in the `_geometry_type_keywords`
```suggestion
                Sequence(
                    OneOf(*_geometry_type_keywords, "GEOMETRY", "GEOGRAPHY"),
                    Sequence(
                        Ref("CommaSegment"),
                        Ref("NumericLiteralSegment"),
                        optional=True,
                    ),
                ),
```

Reply:
My bad. I refactored now.